gradle.properties

```diff
@@ -13,7 +13,7 @@
 #
 
 group=com.huawei.cloud
-version=0.12.0-SNAPSHOT
+version=0.12.0
```

Comment:
this should be made by the release workflow. I guess it's related by the fact that the edc-build plugin is not available anymore at that snapshot version, let's try to do like this and see how it behaves, it could work anyway out of the box